src/app/Components/Home/Home-Components/home/home.component.html

```diff
@@ -7,7 +7,7 @@
       <app-messages-container></app-messages-container>
     </div>
   </div>
-
+  <mat-divider style="width: 100%"></mat-divider>
```

Comment:
why inline style thought...

Reply:
it is better readable than using a class for this simple property, because in the html you don't know what the class do.